ASTProcessor/build.gradle.kts

```diff
@@ -23,6 +23,8 @@ dependencies {
 	testImplementation(libs.springboot.starter.test)
 	testRuntimeOnly(libs.junit.launcher)
 	antlr(libs.antlr)
+	implementation("org.slf4j:slf4j-api:2.0.16")
```

Comment:
put these dependencies inside the `Argos\gradle\libs.versions.toml` file

Reply:
done